lib/momento/simple_cache_client.rb

```diff
@@ -6,10 +6,14 @@
 require_relative 'exceptions'
 
 module Momento
+  # rubocop:disable Metrics/ClassLength
```

Comment:
What does this do?

Reply:
Rubocop is a Ruby style checker. One of the things it checks is that classes aren't too long. This disables that check for SimpleCacheClient. It's a bit too long, but we're not ready to fix that.